docs/package.json

```diff
@@ -33,7 +33,7 @@
     "gray-matter": "^4.0.3",
     "lodash": "^4.17.21",
     "mdx-prism": "^0.3.3",
-    "next": "^13.5.5",
+    "next": "^14.1.1",
```

Comment:
Upgrading docs to 14 seems a little scary to do here? Do we know what's changing from 13 - 14?

Reply:
Yeah that's fair, I looked at the [release notes](https://github.com/vercel/next.js/releases/tag/v14.0.0) but there's definitely a lot of changes. If we're not comfortable upgrading right now we could downgrade to <13.4.0.